### PR TITLE
work with component, structure and rename some thing

### PR DESCRIPTION
**Pull request by BorodinDemid**

### CODE REVIEW

File: site/src/components/BenchmarkResults.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const BenchmarkResults = ({data}) => {
+    return (
+        <div>
+            <h1>Node package manager benchmark</h1>
+            <hr/>
+            <p>This benchmark compares the performance of <a href="https://github.com/npm/npm">npm</a>, <a
+                href="https://github.com/rstacruz/pnpm">pnpm</a> and <a href="https://github.com/yarnpkg/yarn">yarn</a>.
+            </p>
+            {data.map(singleBenchmark => <FrameworkTable data={singleBenchmark} key={singleBenchmark.id}/>)}
```
Comment (iamstarkov) [inline site/src/components/BenchmarkResults.js]: lets simplify things a bit more.

BenchmarkResults contains of header and FrameworkTable's map. Id suggest eject last bit from BenchmarkResults and inject FrameworkTable's map straight into App

File: site/src/App.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-
+import FrameworkTable from './components/FrameworkTable';
+import Header from './components/Header';
 
 const App = ({data}) => {
     return (
         <div>
-            <BenchmarkResults data={ data }/>
+            <Header/>
+            {data.map(singleBenchmark => <FrameworkTable data={singleBenchmark} key={singleBenchmark.id}/>)}
```
Comment (iamstarkov) [inline site/src/App.js]:
```diff
-            {data.map(singleBenchmark => <FrameworkTable data={singleBenchmark} key={singleBenchmark.id}/>)} 
+            { data.map(item =>
+                <FrameworkTable data={singleBenchmark} key={singleBenchmark.id}/>
+            )} 
```

lets aim readability as well

File: site/src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const BenchmarkResults = () => {
```
Comment (iamstarkov) [inline site/src/components/Header.js]: r u sure its still `BenchmarkResults`?

File: site/src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    return (
+        <div>
+            <h1>Node package manager benchmark</h1>
+            <hr/>
+            <p>This benchmark compares the performance of <a href="https://github.com/npm/npm">npm</a>, <a
+                href="https://github.com/rstacruz/pnpm">pnpm</a> and <a href="https://github.com/yarnpkg/yarn">yarn</a>.
+            </p>
+        </div>
+    )
+};
```
Comment (iamstarkov) [inline site/src/components/Header.js]: lets redo it this way:

```jsx
const One = () => (
  <div>
  </div>
)
```

Comment (iamstarkov) [inline site/src/components/Header.js]: in App as well

Comment (iamstarkov) [inline site/src/components/Header.js]: and FrameworkTable

File: site/src/components/FrameworkTable.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            <tr>
+                <td>{data.commandYarn}</td>
+                <td>{data.timeYarn}</td>
+                <td>{data.sizePnpm}</td>
+            </tr>
+            <tr>
+                <td>{data.commandPnpm}</td>
+                <td>{data.timePnpm}</td>
+                <td>{data.sizePnpm}</td>
+            </tr>
```
Comment (iamstarkov) [inline site/src/components/FrameworkTable.js]: maybe wrap it in map as well?

```jsx
['Npm', 'Yarn', 'Pnpm'].map(item => `command${item}`).map(item => (
    <tr>
        <td>{ data[item] }</td>
        <td>{ data.timePnpm }</td>
        <td>{ data.sizePnpm }</td>
    </tr>
))
```

**Comment by BorodinDemid** [comment]
zaebca)

**Pull request merged**